client/app/scripts/utils/contrast-utils.js

```diff
@@ -1,7 +1,22 @@
-export const contrastModeUrl = 'contrast.html';
+/* eslint-disable no-underscore-dangle */
+/**
+ * Change the Scope UI theme from normal to high-contrast.
+ * This will inject a stylesheet into <head> and override the styles.
+ *
+ * A window-level variable is written to the .html page during the build process that contains
+ * the filename (and content hash) needed to download the file.
+ */
 
-const contrastMode = window.location.pathname.indexOf(contrastModeUrl) > -1;
 
-export function isContrastMode() {
-  return contrastMode;
+export function loadTheme(theme = 'normal') {
+  if (window.__WEAVE_SCOPE_THEMES) {
+    // Load the pre-built stylesheet.
+    const stylesheet = window.__WEAVE_SCOPE_THEMES[theme];
+    const head = document.querySelector('head');
+    const link = document.createElement('link');
+    link.rel = 'stylesheet';
+    link.href = `${window.__WEAVE_SCOPE_THEMES.publicPath}${stylesheet}`;
+
+    head.appendChild(link);
```

Comment:
It would be nice if we'd also remove the previous theme here, as right now every toggle adds one `<link>` tag to the html, so by toggling a lot we end up with something like this:

```
<link rel="stylesheet" href="style-contrast-theme-fd61e9572aa66a409ca1.css?c044bfbe96884908980f">
<link rel="stylesheet" href="style-app-4134b6394afe6c264f1e.css?c044bfbe96884908980f">
<link rel="stylesheet" href="style-contrast-theme-fd61e9572aa66a409ca1.css?c044bfbe96884908980f">
<link rel="stylesheet" href="style-app-4134b6394afe6c264f1e.css?c044bfbe96884908980f">
<link rel="stylesheet" href="style-contrast-theme-fd61e9572aa66a409ca1.css?c044bfbe96884908980f">
<link rel="stylesheet" href="style-app-4134b6394afe6c264f1e.css?c044bfbe96884908980f">
...
```